kubernetes-manifests/config.yaml

```diff
@@ -39,13 +39,4 @@ data:
   USE_DEFAULT_DATA: "True"
   # All default user accounts are hardcoded to use the login password 'password'
   DEFAULT_LOGIN_PASSWORD: "password"
-  DEFAULT_USER_ACCOUNT: "1033623433"
-  DEFAULT_USER_USERNAME: "testuser"
-  DEFAULT_USER_NAME: "Eve"
-  DEFAULT_DEPOSIT_ACCOUNT: "9099791699"
-  DEFAULT_DEPOSIT_ROUTING: "808889588"
-  DEFAULT_DEPOSIT_LABEL: "External Bank"
-  DEFAULT_CONTACT_ACCOUNT_A: "1044226144"
-  DEFAULT_CONTACT_NAME_A: "Alice"
-  DEFAULT_CONTACT_ACCOUNT_B: "1055757655"
-  DEFAULT_CONTACT_NAME_B: "Bob"
+  DEFAULT_LOGIN_USERNAME: "eve"
```

Comment:
So no more testuser? Isn't Eve usually the evil one?

Reply:
That's fair.  Hmm, maybe testuser can be friends with alice, bob, and eve?

Reply:
That sounds good to me. 3 contacts might look better in the history list too, more variety. If it's a lot of work, this isn't a big issue for me though

Reply:
Done and done.  Easy peasy with these new script files.